flumine/markets/market.py

```diff
@@ -62,6 +62,13 @@ def close_market(self) -> None:
             extra=self.info,
         )
 
+    def belongs_to_strategy(self, strategy) -> bool:
```

Comment:
Sure, but is it not the other way around, that a strategy is subscribed to a market? How about `is_a_subscriber(strategy)`?

We could also remove this method and simply do `market.market_book.streaming_unique_id in strategy.stream_ids`. It would be a tiny bit faster because `streaming_unique_id` could be cached instead of looked up for every stategy.

Reply:
This method could be removed because it is not essential for `_process_market_books` check, and in the context of `_process_market_catalogues`, we could use a method I proposed below.

Reply:
This method has been removed in the latest commit.